### ISSUE TITLE
[WCAG] Layers Panel - Create keyboard flow when Show in Time Slider button pressed

### ISSUE THREAD

**Issue by kenchase** [Canadian-Geospatial-Platform/geoview#3244]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

In WCAG mode (keyboard):

1. Open the Layers panel
2. Navigate to a layer panel that includes a Time Slider button
3. Press the Time Slider button
4. Focus sequence is lost

### Expected Behavior

In WCAG mode (keyboard):

Need to work out details, but the high-level flow should be:

1. Open the Layers panel
2. Navigate to a layer panel that includes a Time Slider button
3. Press the Time Slider button
4. Time Slider Exit button receives focus
